development/workflow.rst

```diff
@@ -0,0 +1,667 @@
+   At the start of each cycle, your T/CAM will create an epic (or several) and allocate *story points* to that epic.
+   You don't work directly on an epic; rather you work on *stories* (below) that cumulatively accomplish the epic.
+
+.. _workflow-jira-issues:
+
+Tickets
+-------
+
+All development work is done on these three types of **JIRA issues** that are generically referred to as **tickets**:
+
+Story
+   Stories are issues associated with an epic.
+   That is, stories are for work that accomplish your main goals for a cycle.
+Bug
+   A bug is an emergent (not planned with an epic) ticket that fixes a fault in the software that exists on ``master``.
+   Bug tickets are not associated with an epic.
+Enhancement
+   An enhancement is essentially a feature request.
+   Like a *bug*, an enhancement is emergent.
+   Unlike a bug, an enhancement adds new functionality.
+   Enhancements differ from stories in that they have no epic link.
+
+Issue semantics were discussed in `RFC-43 <https://jira.lsstcorp.org/browse/RFC-43>`_.
+
+As a developer, you can create tickets to work on.
+You can also create bug or enhancement tickets and assign them to others (ideally with some consultation).
+
+.. _workflow-jira-ticket-creation:
+
+Creating a ticket
+-----------------
+
+You can create a ticket from the `JIRA web app <https://jira.lsstcorp.org>`_ toolbar using the **Create** button.
+For more general information, you can consult the `LSST JIRA wiki <https://confluence.lsstcorp.org/display/JIRAatLSSTUserGuide/JIRA+at+LSST+User%27s+Guide+Home>`_ and Atlassian's docs for `JIRA <https://confluence.atlassian.com/jirasoftwarecloud/jira-software-documentation-764477791.html>`_ and `JIRA Agile <https://confluence.atlassian.com/agile067>`_.
+
+JIRA allows a myriad of metadata to be specified when creating a ticket; these are the most relevant fields:
+
+Project
+   This should be set to **Data Management**, unless you are creating a ticket for a different LSST subsystem.
+Issue Type
+   If the work is associated with an epic, the issue type is a 'Story.'
+   For emergent work, 'Bug' or 'Enhancement' can be used (see above for semantics).
+Summary
+   This is the ticket's title and should be written to help colleagues browsing JIRA dashboards.
+Components
+   You should choose from the pre-populated list of components to specify what part of the DM system the ticket relates to.
+   If in doubt, ask your T/CAM.
+Assignee
+   Typically you will assign yourself (or your T/CAM will assign you) to a ticket.
+   You can also assign tickets to others.
+   If you are uncertain about who the assignee should be you can allow the ticket to be automatically assigned (which defaults to the component's T/CAM; `RFC-51 <https://jira.lsstcorp.org/browse/RFC-51>`_).
+Description
+   For stories, you should specify the ticket's scope and outline your implementation design in this field.
```

Comment:
Ideally there should be a clear description of the deliverable that can serve as a "definition of done".